docs/user-guide/ze-working-with-commands.md

```diff
@@ -0,0 +1,51 @@
+# Working with commands
+
+You can issue the same commands used in ISPF with Zowe Explorer.
+
+MVS commands can be used to obtain information from the zOS console, while SSH commands can be used to issue shell commands to Unix System Services (USS). TSO commands can be used to invoke mainframe scripts in a zOS terminal session.
+
+## Issuing MVS commands
+
+1. Expand **DATA SETS**, **UNIX SYSTEM SERVICES (USS)**, or **JOBS** in the **Side Bar**.
+2. Right-click on your profile and select the **Issue MVS Command** option.
+
+   Alternatively, press the `F1` key to open the **Command Pallette**, and then select the **Zowe Expolorer: Issue MVS Command** option.
+
+3. In the **Quick Pick**, enter a new command or select a saved command.
+4. Press the `Enter` key to execute the command.
+
+    The output displays in the **Output** panel.
+
+> Example command: Issue `profile` to get information on your zOS system user profile.
+
+   ![Issue a MVS command](../images/ze/ZE-Jobs-Issue-TSO-Command.gif)
+
+## Issuing SSH commands
+
+1. Expand **DATA SETS**, **UNIX SYSTEM SERVICES (USS)**, or **JOBS** in the **Side Bar**.
+2. Right-click on your profile and select the **Issue Unix Command** option.
+
+   Alternatively, press the `F1` key to open the **Command Pallette**, then select the **Zowe Explorer: Issue Unix Command** option.
+
+3. In the **Quick Pick**, enter a new command or select a saved command.
+4. Press the `Enter` key to execute the command.
+
+   The output displays in the **Output** panel.
+
+> Example command: Issue `ls` to list the contents of the current directory.
+
+## Issuing TSO commands
+
+1. Expand **DATA SETS**, **UNIX SYSTEM SERVICES (USS)**, or **JOBS** in the **Side Bar**.
+2. Right-click on your profile and select the **Issue TSO Command** option.
+
+   Alternatively, press the `F1` key to open the **Command Pallette**, then select the **Zowe Explorer: Issue TSO Command** option.
+
+3. In the **Quick Pick**, enter a new command or select a saved command.
+4. Press the `Enter` key to execute the command.
+
+   The output displays in the **Output** panel.
+
+   ![Issue a TSO command](../images/ze/ZE-TSO-Command.gif)
+
+> Example command: Issue `display time` to get the system time.
```

Comment:
`display time` is not a valid TSO command. We could change the command to maybe `LISTCAT` or `TIME` for something valid but generic.

Reply:
@t1m0thyj - what are your thoughts?

Reply:
See above - `profile` is a TSO command not an MVS command, sorry for the confusion 😋